packages/@uppy/companion/src/server/provider/dropbox/index.js

```diff
@@ -73,6 +73,7 @@ class DropBox extends Provider {
         prefixUrl: 'https://content.dropboxapi.com/2',
         headers: {
           'Dropbox-API-Arg': httpHeaderSafeJson({ path: String(id) }),
+          Connection: 'keep-alive', // important because https://github.com/transloadit/uppy/issues/4357
```

Comment:
We only want this for Dropbox? No other providers?

Reply:
dropbox is the one with the problem. i thought it's safer not to change it for more providers than we need to.
see discussion here: https://github.com/transloadit/uppy/issues/4357